src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.hpp

```diff
@@ -102,6 +102,8 @@ class LLMCompiledModel : public ov::npuw::ICompiledModel {
     int32_t m_gemma_sliding_window_size = 0;
 
     bool m_is_whisper = false;
+
+    ov::AnyMap m_model_rt_info;
```

Comment:
We are now considering all newly added fields to the `LLMCompiledModel` as the fields that participate in serialization/deserialization in export/import and caching activities. However, I am not sure about this field, it needs to be checked how to handle it for import and caching.